### PR TITLE
bump ops version

### PR DESCRIPTION
**Pull request by beliaev-maksim**

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-ops
+ops>2
 oci-image
-serialized-data-interface<0.7
+serialized-data-interface==0.6.0
```
Comment (ca-scribner) [inline requirements.in:6]: Is there a reason why we'd pin this specifically?  Does <0.7 give troubles?

Comment (ca-scribner) [reply requirements.in:6]: I ask mainly because we've been bitten by having such strict pinning in the past

File: requirements.in [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-ops
+ops>2
```
Comment (ca-scribner) [inline requirements.in:4]: Is >2 needed here?  I don't see anything we're changing that forces `>2`

If we did not have `>2` here, wouldn't we still get the same version of ops?

**Review by ca-scribner — CHANGES_REQUESTED** [review]
Added a few minor comments, but the main thing here is that integration tests are failing.  Perhaps something is not backward compatible?

**Comment by beliaev-maksim** [comment]
closed in favor of SDI deletion

**Pull request closed without merging**